extension/package.json

```diff
@@ -12,7 +12,10 @@
     "check": "nsp check",
     "cover:unit": "nyc _mocha --require ../test/setupBackend.js --recursive ../test/unit",
     "cover:integration": "nyc _mocha --use_strict --timeout 3000 --require ../test/setupBackend.js --recursive ../test/integration",
-    "cover": "nyc _mocha --use_strict --timeout 3000 --require ../test/setupBackend.js --recursive ../test/integration --recursive ../test/unit"
+    "cover": "nyc _mocha --use_strict --timeout 3000 --require ../test/setupBackend.js --recursive ../test/integration --recursive ../test/unit",
+    "update:schema": "npm run schema:fetch && npm run schema:transform-to-types && eslint --fix lib/types.js",
```

Comment:
We could reverse script to be consistent with other schema scripts: schema:update